thirdparty/DiskANN/src/pq_flash_index.cpp

```diff
@@ -53,49 +54,12 @@
   ((((_u64) (id)) % nvecs_per_sector) * data_dim * sizeof(float))
 
 namespace {
-  void aggregate_coords(const unsigned *ids, const _u64 n_ids,
-                        const _u8 *all_coords, const _u64 ndims, _u8 *out) {
-    for (_u64 i = 0; i < n_ids; i++) {
-      memcpy(out + i * ndims, all_coords + ids[i] * ndims, ndims * sizeof(_u8));
-    }
-  }
-
-  void pq_dist_lookup(const _u8 *pq_ids, const _u64 n_pts,
-                      const _u64 pq_nchunks, const float *pq_dists,
-                      float *dists_out) {
-#if defined(__ARM_NEON__) || defined(__aarch64__)
-    __builtin_prefetch((char *) dists_out, 1, 3);
-    __builtin_prefetch((char *) pq_ids, 0, 3);
-    __builtin_prefetch((char *) (pq_ids + 64), 0, 3);
-    __builtin_prefetch((char *) (pq_ids + 128), 0, 3);
-#else
-    _mm_prefetch((char *) dists_out, _MM_HINT_T0);
-    _mm_prefetch((char *) pq_ids, _MM_HINT_T0);
-    _mm_prefetch((char *) (pq_ids + 64), _MM_HINT_T0);
-    _mm_prefetch((char *) (pq_ids + 128), _MM_HINT_T0);
-#endif
-    memset(dists_out, 0, n_pts * sizeof(float));
-    for (_u64 chunk = 0; chunk < pq_nchunks; chunk++) {
-      const float *chunk_dists = pq_dists + 256 * chunk;
-      if (chunk < pq_nchunks - 1) {
-#if defined(__ARM_NEON__) || defined(__aarch64__)
-        __builtin_prefetch((char *) (chunk_dists + 256), 0, 3);
-#else
-        _mm_prefetch((char *) (chunk_dists + 256), _MM_HINT_T0);
-#endif
-      }
-      for (_u64 idx = 0; idx < n_pts; idx++) {
-        _u8 pq_centerid = pq_ids[pq_nchunks * idx + chunk];
-        dists_out[idx] += chunk_dists[pq_centerid];
-      }
-    }
-  }
   constexpr _u64 kRefineBeamWidthFactor = 2;
   constexpr _u64 kBruteForceTopkRefineExpansionFactor = 2;
-  auto calcFilterThreshold = [](const auto topk) -> const float {
+  auto           calcFilterThreshold = [](const auto topk) -> const float {
```

Comment:
What is this change🤔

Reply:
i run pre-commit run， and code format change.